src/pages/[platform]/build-a-backend/add-aws-services/analytics/app-uninstall/index.mdx

```diff
@@ -0,0 +1,40 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Uninstalling the app',
+  description:
+    'Understand how to handle persistent data on a device when a user uninstalls the app.',
+  platforms: ['swift', 'android'],
+  canonicalUrl: '/[platform]/build-a-backend/add-aws-services/analytics/app-uninstall/'
```

Comment:
For now, I think you can remove the `canonicalUrl` property.

This is so that we can specify a platform as the canonical URL for SEO purposes. You would need to specify a platform if both of these platform pages are duplicates
```suggestion
  platforms: ['swift', 'android']
```